Description: Implement Keyword key in the .desktop file
Author: Luca Falavigna <dktrkranz@debian.org>

Index: gtg-0.3.1/gtg.desktop
===================================================================
--- gtg-0.3.1.orig/gtg.desktop	2013-12-12 11:14:33.000000000 +0000
+++ gtg-0.3.1/gtg.desktop	2013-12-12 11:15:23.147274875 +0000
@@ -10,6 +10,7 @@
 Terminal=false
 Type=Application
 Categories=Office;ProjectManagement;
+Keywords=Organizer;Task;Activity;Plan;Time;
 MimeType=x-scheme-handler/gtg;
 StartupNotify=true
 X-Ayatana-Desktop-Shortcuts=NewTask
